### PR TITLE
Journal week 4

### PR DESCRIPTION
**Pull request by Ricecoder**

### CODE REVIEW

**Comment by cmauban** [comment]
Nice start!! Ahhh yeah we destroyed our beautiful blueprints :( That tutorial will help out a lot! :+1:

**Comment by nickycadavillo** [comment]
:+1:  Definitely need a tutorial on yo; looking forward to that! Your journal is off to a great start. Things definitely got REAL last week. Keep swimmin'!

**Comment by jeremyrist** [comment]
:+1: Nice tutorial!

File: journal-4.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##Week 4 AKA The week things got real!
+
+Week 4 we took all of our beautiful Blueprint sites that we had worked so hard on and DESTROYED them. We converted our css to sass, rebuilt the html, and converted the javascript to jquery.
```
Comment (kellymurray) [inline journal-4.md:3]: :fire: the blueprints!!

**Pull request merged**